api/services/GithubSearch.js

```diff
@@ -0,0 +1,80 @@
+const github = require('octonode');
+const _ = require('lodash');
+
+const client = github.client(sails.config.statos.githubAccessToken);
+
+const tajawalSearch = client.search();
+
```

Comment:
it's better to class a class and inject these services into the GithubSearch class. better for testing too